examples/mspaint.py

```diff
@@ -8,15 +8,36 @@
 The example shows how to work with MS Paint application. It opens
 JPEG image and resizes it using "Resize and Skew" dialog.
 """
-
+import logging
```

Comment:
If you use `print` function please do `from __future__ import print_function` as a first import to be Py2/Py3 compatible.